authz.go

```diff
@@ -129,6 +129,8 @@ const (
 	NotificationEndpointResourceType = ResourceType("notificationEndpoints") // 15
 	// ChecksResourceType gives permission to one or more Checks.
 	ChecksResourceType = ResourceType("checks") // 16
+	// DBRPType gives permission to one or more DBRPs.
+	DBRPType = ResourceType("dbrp") // 16
```

Comment:
@gianarb could this be `DBRPResourceType`?

Just for conformity with the others